operator/internal/validation/lokistack_test.go

```diff
@@ -390,6 +390,240 @@ var ltt = []struct {
 			},
 		),
 	},
+	{
+		desc: "enabling global limits OTLP IgnoreDefaults without resource attributes",
+		spec: lokiv1.LokiStack{
+			Spec: lokiv1.LokiStackSpec{
+				Storage: lokiv1.ObjectStorageSpec{
+					Schemas: []lokiv1.ObjectStorageSchema{
+						{
+							Version:       lokiv1.ObjectStorageSchemaV11,
+							EffectiveDate: "2020-10-11",
+						},
+						{
+							Version:       lokiv1.ObjectStorageSchemaV12,
```

Comment:
This does not impact the test, but "logically" this schema configuration is invalid for OTLP, because there is no V13 schema. Same for the other tests.